DemoPage/examples/Checkbox/index.js

```diff
@@ -10,9 +10,9 @@ module.exports = React.createClass({
 
   t: function(name) {
     var translation = {
-      join_label: 'Join or not',
-      terms_label: 'Terms & Conditions',
-      opt_in_label: 'Opt in'
+      'join_label': 'Join or not',
```

Comment:
key identifiers should use camelCase, but since this pattern was strewn throughout the whole of HUI, I just stringified them to pass the linting. I've since removed that linting rule because clearly we don't care about proper casing in HUI.